packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/WholeMeetingSummaryResult.tsx

```diff
@@ -93,7 +93,7 @@ const WholeMeetingSummaryResult = ({meetingRef}: Props) => {
             <td
               align='center'
               style={textStyle}
-              className='summary-link-style'
```

Comment:
I updated this as link-style is being used in two places now, not just the summary